airflow-core/src/airflow/serialization/serialized_objects.py

```diff
@@ -2346,7 +2346,7 @@ def _create_orm_dagrun(
     run.dag = dag
     # create the associated task instances
     # state is None at the moment of creation
-    run.verify_integrity(session=session, dag_version_id=dag_version.id)
+    run.verify_integrity(session=session, dag_version_id=dag_version.id if dag_version else None)
```

Comment:
Do we need to run verify integrity at all if dag version is none? (I dont have the context loaded in my head right now)

Reply:
Yes. We need to create the Task Instances